api/tests/test_account_views.py

```diff
@@ -31,18 +37,24 @@ def test_account_detail_no_account(api_client):
     assert response.status_code == 404
 
 
-def test_account_put_invalid(ratom_file, api_client):
+def test_account_put_invalid(ratom_file, api_client, file_serializer_validation_mock):
     # Test PUT with invalid serializer
     url = reverse("account_detail", args=[ratom_file.account.pk])
     data = {"filename": "x" * 201}
+    instance = file_serializer_validation_mock.return_value
```

Comment:
This makes the tests pass, but feels like it's mixing two test cases to me. The ``"x" * 201`` was to trigger the filename too long validation error. I think the ``instance.exists = False`` should be a separate test case.

Reply:
:+1:

Reply:
I don't think this needs to be two tests, but it should be True and not False, for this test. The False was hiding the bad validation on the filename.